### PR TITLE
👮🏽 Rubocop config 👮🏽 #484

### LINKED ISSUE #484: 👮🏽 Rubocop config 👮🏽
Screen Shot 2020-05-31 at 1 55 10 am

### PR DESCRIPTION
**Pull request by tHe-AK**
Fixes #484 
- Added missing rubocop configs & disabled few rules.
- Fixed 'Style/StringConcatenation' rule errors

### CODE REVIEW

**Comment by jbampton** [comment]
Looks like the build failed here -> https://app.circleci.com/pipelines/github/slurpcode/slurp/1363/workflows/a6be8cb7-3315-48b9-b943-ad87ac2fb499/jobs/5201  you will need to fix that.

File: .rubocop.yml [hunk 1]
```diff
@@ -51,3 +66,142 @@ Style/MutableConstant:
...
+  Enabled: true
+
+Lint/UnreachableLoop:
+  Enabled: true
+
+Lint/UselessMethodDefinition:
+  Enabled: true
+
+Lint/UselessTimes:
+  Enabled: true
```
Comment (jbampton) [inline .rubocop.yml:207]:
```suggestion
  Enabled: true

```

**Review by jbampton — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (tHe-AK) [reply .rubocop.yml:207]: https://app.circleci.com/pipelines/github/slurpcode/slurp/1364/workflows/6c16fe9b-bb59-463a-9b17-f0fc1874b71c

**Comment by jbampton** [comment]
Needs a rebase

**Comment by jbampton** [comment]
Why did you close this ?

**Comment by tHe-AK** [comment]
I have deleted the source branch and created a new branch with same name to have a clean git history. I was not aware that deleting source will close the PR, as this is not the behaviour in BitBucket.

**Comment by jbampton** [comment]
If you had rebased like I said you would have a clean git history.  Or you can squash your commits.

**Pull request closed without merging**